clang/tools/c-index-test/core_main.cpp

```diff
@@ -1358,26 +1358,26 @@ int indextest_core_main(int argc, const char **argv) {
                                       : Optional<std::string>(options::CASPath);
 
   if (options::Action == ActionType::ScanDeps) {
-    if (options::InputFiles.empty()) {
+    if (options::WorkingDir.empty()) {
       errs() << "error: missing working directory\n";
```

Comment:
Nit: how about "missing -working-dir <cwd>"; same below.

Reply:
Done in https://github.com/apple/llvm-project/pull/7295/commits/da67386316bc111633977b57be7e051c12ddc7a1